build_test_publish_images.py

```diff
@@ -188,21 +213,32 @@ def main(build_dir, tf_dir, dry_run, tests, publish_step):
         ami = yaml.load(f)['us-west-2']
     terraform_add_os(build_dir, tf_dir, platform, vars_string, ami, os_name)
     shutil.copyfile(cluster_profile, os.path.join(tf_dir, 'desired_cluster_profile.tfvars'))
+    dcos_version = build_dir.split('/')[3].split('-')[1]
+    dcos_major_version = dcos_version[0:4] if not dcos_version.isalpha() else 'master'
+    url = "https://downloads.dcos.io/dcos/{}/dcos_generate_config.sh"
+    dcos_download_url = url.format('stable/' + dcos_version) if not dcos_version.isalpha() else url.format('testing/' + dcos_version)
+    with open(os.path.join(tf_dir, 'desired_cluster_profile.tfvars'), "a") as f:
+       f.write('\ndcos_version = "{}"\n'.format(dcos_version))
+       f.write('custom_dcos_download_path = "{}"\n'.format(dcos_download_url))
```

Comment:
Please add this new block of code in a new method.
like write_dcos_version_to_cluster_profile()